src/main/java/org/jadice/filetype/matchers/EncryptedZIPMatcher.java

```diff
@@ -1,19 +1,20 @@
 package org.jadice.filetype.matchers;
 
 import java.io.IOException;
-import java.util.zip.ZipException;
-import java.util.zip.ZipInputStream;
 
 import org.jadice.filetype.Context;
 import org.jadice.filetype.io.SeekableInputStream;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import net.lingala.zip4j.exception.ZipException;
+import net.lingala.zip4j.io.inputstream.ZipInputStream;
+
 public class EncryptedZIPMatcher extends Matcher {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(EncryptedZIPMatcher.class);
 
-  private static final String NEEDLE = "encrypted ZIP".toLowerCase();
+  private static final String NEEDLE = "Wrong password".toLowerCase();

```

Comment:
That the needle text has changed is correct, right?

Reply:
Yes, that's correct.